test/metabase/query_processor_test/date_time_zone_functions_test.clj

```diff
@@ -778,7 +780,35 @@
                          :expressions
                          {"tz,dt" [:datetime-diff $dt_tz $dt :second]
                           "tz,d"  [:datetime-diff $dt_tz $d :second]}})
-                      (mt/formatted-rows [int int int])
+                      (mt/formatted-rows [int int])
+                      first))))))))
+
+(deftest athena-datetime-diff-mixed-types-test
```

Comment:
Same as this, but for Athena https://github.com/metabase/metabase/blob/0070c885f87a15872862196cf1ec677fed093189/test/metabase/query_processor_test/date_time_zone_functions_test.clj#L769

https://github.com/metabase/metabase/blob/0070c885f87a15872862196cf1ec677fed093189/test/metabase/query_processor_test/date_time_zone_functions_test.clj#L769